texts/depositing/compliance-notice.md

```diff
@@ -1,9 +1,5 @@
 ---
-title: Compliance
+title: Data Overview
```

Comment:
Americans prefer _Data Overview_ (each word capital).  
British prefer _Data overview_ (sentence case).

CORE is where knights and queen, we prefer British.